src/main/resources/mappingio/lang/en_us.properties

```diff
@@ -0,0 +1,14 @@
+format.tiny_file = Tiny File
```

Comment:
Most of this shouldnt even be translated, other than maybe "File". Im not really sure this should be handled in mapping-io.

Reply:
Some languages will need to hyphenate individual words, switch their order around, or even outright translate the format names. For example in German, you'd have `Tiny-v2-Datei` and `IntelliJ-Migrationszuordnungsdatei` (at least when going with the German language pack's translation).

I think this makes sense being part of Mapping-IO natively, I don't want to duplicate the translation files across every GUI application making use of MIO: Enigma, Matcher, JADX, just to name a few. And you're not forced to use MIO's translations either, you can always delegate to a custom localization system via `Translatable#translationKey()`. Not to mention that these facilities are required for #88 and #94 to provide user-friendly warnings and error messages in Enigma.